components/ProductForm.tsx

```diff
@@ -169,49 +172,37 @@ export default function ProductForm({
               onChange={handleChangeFile}
             />
           </Button>
-          <Button variant="outlined" onClick={handleResetImage}>
-            Reset
-          </Button>
+          {imageSource !== initialData.image && (
```

Comment:
I’ve just noticed that when opening the form in “add product” mode, we initially show the reset button because the condition `imageSource !== initialData.image` is true (`imageSource` is an empty string and `initialData.image` is undefined). So we could adjust that condition.
```suggestion
          {imageSource && imageSource !== initialData.image && (
```